test/java/seedu/addressbook/logic/LogicTest.java

```diff
@@ -455,6 +455,27 @@ public void execute_find_matchesIfAnyKeywordPresent() throws Exception {
                                 true,
                                 expectedList);
     }
+    @Test
+    public void execute_findtag_matchesIfAnyKeywordPresent() throws Exception {
```

Comment:
Good that you have added a test case. In future try to add multiple test cases. e.g. multiple existing tags, invalid tag, valid and invalid tag combination etc.